app/src/main/java/org/fossasia/openevent/app/common/data/models/Event.java

```diff
@@ -47,55 +67,84 @@
 
     @Id(LongIdHandler.class)
     @PrimaryKey
-    public long id;
+    public Long id;
     public String paymentCountry;
     public String paypalEmail;
     public String thumbnailImageUrl;
     public String schedulePublishedOn;
     public String paymentCurrency;
     public String organizerDescription;
-    public boolean isMapShown;
     public String originalImageUrl;
     public String onsiteDetails;
     public String organizerName;
-    public boolean canPayByStripe;
     public String largeImageUrl;
     public String timezone;
-    public boolean canPayOnsite;
     public String deletedAt;
     public String ticketUrl;
-    public boolean canPayByPaypal;
     public String locationName;
-    public boolean isSponsorsEnabled;
-    public boolean hasOrganizerInfo;
-    public boolean isSessionsSpeakersEnabled;
     public String privacy;
     public String codeOfConduct;
     public String state;
-    public double latitude;
-    public String startsAt;
     public String searchableLocationName;
-    public boolean isTicketingEnabled;
-    public boolean canPayByCheque;
     public String description;
     public String pentabarfUrl;
     public String xcalUrl;
     public String logoUrl;
     public String externalEventUrl;
-    public boolean isTaxEnabled;
     public String iconImageUrl;
     public String icalUrl;
     public String name;
-    public boolean canPayByBank;
-    public String endsAt;
     public String createdAt;
-    public double longitude;
     public String bankDetails;
     public String chequeDetails;
     public String identifier;
 
     public boolean isComplete;
 
+    @JsonSerialize(using = ToStringSerializer.class)
+    public double latitude;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public double longitude;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean canPayByStripe;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean canPayByCheque;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean canPayByBank;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean canPayByPaypal;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean canPayOnsite;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean isSponsorsEnabled;
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean hasOrganizerInfo;
+
+    @JsonProperty("is-sessions-speakers-enabled")
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean isSessionsSpeakersEnabled;
+    @JsonProperty("is-ticketing-enabled")
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean isTicketingEnabled;
+    @JsonProperty("is-tax-enabled")
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean isTaxEnabled;
+    @JsonProperty("is-map-shown")
+    @JsonSerialize(using = ToStringSerializer.class)
+    public boolean isMapShown;
```

Comment:
This shouldn't be serialized to string. If this is a requirement, it is more likely a bug on server

Reply:
Yes I have to add them. Please see my above comment.

Reply:
Earlier i was getting server error sucht that it was accpting `"false"` and not `false` that's why i had to add it but now I'm not getting any related errors. So I'll remove it then.